heudiconv/convert.py

```diff
@@ -821,6 +841,14 @@ def save_converted_files(res, item_dicoms, bids_options, outtype, prefix, outnam
         echo_times = sorted(echo_times)  # also converts to list
         channel_names = sorted(channel_names)  # also converts to list
 
+        iops = sorted(list(set(
+            str(b.get('ImageOrientationPatientDICOM', ''))
+            for b in bids_metas
+            if b
+        )))
+
+        is_multiorient = len(iops) > 1
+        
```

Comment:
forgot to call `update_mutliorient_name` below?

would need also a unittest of that function (seems should be doable without files) and ideally some test files to ensure that use here works out